### ISSUE TITLE
Remove Thread From Gui on Bounty Create Flow

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#2350]

**Comment by dmtrjsg** [comment]
After discussing with @WRadoslaw that ID is not available until bounty is actually created, agreed to go with Bounty Title. And in case it is duplicate, add an ordinal number suffix, e.g. "1",

**Comment by dmtrjsg** [comment]
After speaking again and in pursuit of expedience made a decision to use the naming mask of _**bounty_title by member_handle**_ (left fetching the list and validation out of scope)

**Comment by dmtrjsg** [comment]
- [ ] If no category exists, do not pass the ID. If thread ID does not exist, there will be no discussion.
- [ ] Unblock the flow blockage based on category checks. It shall be possible to create it w/o thread.

**Comment by dmtrjsg** [comment]
As of right now, if there is no thread ID we don't show the thread. There is no reason to block the flow since this param is options. @dmtrjsg  to define what and when to do..
